Example/Build-V3-Plugin-Monitoring-ios/GoogleService-Info.plist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>API_KEY</key>
+	<string>AIzaSyB1BnsC_msmalGkBtk7fa0qz06_7_Hf024</string>
+	<key>GCM_SENDER_ID</key>
+	<string>1075568137516</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>com.honest.staging</string>
+	<key>PROJECT_ID</key>
+	<string>the-honest-company</string>
+	<key>STORAGE_BUCKET</key>
+	<string>the-honest-company.appspot.com</string>
```

Comment:
FYI, it's not going to work for the Example app since we are using the wrong GoogleService-Info.plist file.